packages/etherlime-lib/src/types/types.ts

```diff
@@ -0,0 +1,30 @@
+import Global = NodeJS.Global;
+import { Wallet } from "ethers";
+
+export type TxParams = {
+    gasPrice?: number,
+    gasLimit?: number,
+    chainId?: number,
+    etherscanApiKey?: string
+}
+
+export type CompiledContract = {
+    abi: any[],
+    bytecode: string,
+    contractName: string,
+    compiler: {version: string}
+}
+
+export type EtherlimeWallet = {
+    secretKey: string,
+	signer: Wallet
+}
+
+export interface JSONRPCGlobal extends Global {
+  coverageSubprovider: any
+  provider: any
+}
+
+export interface Generic<T> {
```

Comment:
General question - If we rewrite the etherlime-utils in typescript, where will this interface end up, here or in the utils?

Reply:
This interface is universal and can be used for any type that needed to be customised. So it's not specially connected with the deployer, thus I think we can move it in 'etherlime-utils'.